packages/cli/tests/testFilter.js

```diff
@@ -0,0 +1,17 @@
+const FILTERED_TESTS = process.platform === 'win32' ? ['fingerprintWatchCommand'] : [];
```

Comment:
Why does this test need to be filtered? If there's something left to fix for this issue, that's fine, but please remember to create another one to keep track of the loose end.

Reply:
This one gives this error in Travis on Windows:

![image](https://user-images.githubusercontent.com/45714532/195657050-9c69070d-3124-402d-a239-23b6bc1efc9e.png)

You can see it in the logs on this build: https://app.travis-ci.com/github/getappmap/appmap-js/jobs/585469980#L220

I'll make a new issue for it.